### PR TITLE
GraphQL: Fix missing schema info on misguiding lang parameter

### PR DESCRIPTION
**Pull request by plyhun**
## Abstract

When a node language is requested, that does not exist, the whole schema info is also missing from the response.

### General

* [x] Added abstract that describes the change
* [x] Added changelog entry to `/CHANGELOG.adoc`
* [x] Ensured that the change is covered by tests
* [x] Ensured that the change is documented in the docs

### On API Changes

* [x] Checked if the changes are breaking or not
* [x] Added GraphQL API if applicable
* [x] Added Elasticsearch mapping if applicable

### CODE REVIEW

File: verticles/graphql/src/main/java/com/gentics/mesh/graphql/type/InterfaceTypeProvider.java [hunk 1]
```diff
@@ -142,31 +143,21 @@ private void setFields(Consumer<GraphQLFieldDefinition.Builder> setField, boolea
...
-			Object source = env.getSource();
-			if (source instanceof NodeContent) {
-				element = ((NodeContent) source).getNode();
-			} else if (source instanceof HibSchemaVersion) {
-				element = ((HibSchemaVersion) source).getSchemaContainer();
-			} else {
-				element = env.getSource();
-			}
-			return element.getUuid();
+			return getMeshCoreElement(env.getSource(), source -> env.getSource()).getUuid();
```
Comment (plyhun) [inline verticles/graphql/src/main/java/com/gentics/mesh/graphql/type/InterfaceTypeProvider.java:146]: Refactoring.

File: verticles/graphql/src/main/java/com/gentics/mesh/graphql/type/InterfaceTypeProvider.java [hunk 2]
```diff
@@ -142,31 +143,21 @@ private void setFields(Consumer<GraphQLFieldDefinition.Builder> setField, boolea
...
 		}));
 
 		// .etag
 		setField.accept(newFieldDefinition().name("etag").description("ETag of the element").type(GraphQLString).dataFetcher(env -> {
 			GraphQLContext gc = env.getContext();
-			HibTransformableElement<?> element = env.getSource();
-			return element.getETag(gc);
+			return getMeshCoreElement(env.getSource(), unused -> env.getSource()).getETag(gc);
```
Comment (plyhun) [inline verticles/graphql/src/main/java/com/gentics/mesh/graphql/type/InterfaceTypeProvider.java:152]: Crash fix: NodeContent cannot be cast to HibTransformableElement.

File: tests/tests-core/src/main/resources/graphql/node-schema-languages-query [hunk 3]
```diff
@@ -0,0 +1,49 @@
+{
+	exists: node(uuid:"1234567890abcdef1234567890abcdef", lang:["en"]) {
+		# [$.data.exists.availableLanguages.size()=2]
+		availableLanguages
+		schema {
+			# [$.data.exists.schema.uuid=<is-uuid>]
```
Comment (plyhun) [inline tests/tests-core/src/main/resources/graphql/node-schema-languages-query:6]: Schema is loaded.

File: tests/tests-core/src/main/resources/graphql/node-schema-languages-query [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+		languages {
+			# [$.data.exists.languages[0].uuid=<is-uuid>]
+			# [$.data.exists.languages[1].uuid=<is-uuid>]
+			uuid
+
+			# This will be asserted by a special asserter
+			path
+			language
+		}
+		# [$.data.exists.fields.title=News_2015 english title]
```
Comment (plyhun) [inline tests/tests-core/src/main/resources/graphql/node-schema-languages-query:18]: Content exists.

File: tests/tests-core/src/main/resources/graphql/node-schema-languages-query [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+				title
+			}
+		}
+	}
+	missing: node(uuid:"1234567890abcdef1234567890abcdef", lang:["qq"]) {
+		# [$.data.missing.availableLanguages.size()=2]
+		availableLanguages
+		schema {
+			# [$.data.missing.schema.uuid=<is-uuid>]
+			uuid
```
Comment (plyhun) [inline tests/tests-core/src/main/resources/graphql/node-schema-languages-query:30]: Schema is still loaded.

File: tests/tests-core/src/main/resources/graphql/node-schema-languages-query [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+		languages {
+			# [$.data.missing.languages[0].uuid=<is-uuid>]
+			# [$.data.missing.languages[1].uuid=<is-uuid>]
+			uuid
+
+			# This will be asserted by a special asserter
+			path
+			language
+		}
+		# [$.data.missing.fields=<is-null>]
```
Comment (plyhun) [inline tests/tests-core/src/main/resources/graphql/node-schema-languages-query:41]: No content for the false language.

**Review by npomaroli — APPROVED** [review]

**Pull request merged**